src/task-actions.ts

```diff
@@ -365,7 +365,10 @@ function setRuntimeNetwork(args: TaskArguments, hre: HardhatRuntimeEnvironment)
 
     hre.starknet.network = networkName;
     hre.starknet.networkConfig = networkConfig;
-    console.log(`Using network ${hre.starknet.network} at ${hre.starknet.networkConfig.url}`);
+
+    console.log(
```

Comment:
How did this change? Can you format the code with `npm run format`?

Reply:
i added vmLang version in msg so it format on multiple lines

Reply:
But vmLang is only applicable for integrated-devnet. I'd rather have this info removed from the info message.